dev-packages/application-manager/src/generator/webpack-generator.ts

```diff
@@ -114,7 +114,7 @@ module.exports = {
                 test: /worker-main\\.js$/,
                 loader: 'worker-loader',
                 options: {
-                    filename: 'worker-ext.[fullhash].js'
+                    filename: 'worker-ext.js'
```

Comment:
Nothing is impossible, but since we don't need a worker-loader we need to keep somewhere the file name.
Before it was inside generated worker-loader script
```
  return new Worker(__webpack_require__.p + "worker-ext.5f6918e41dd2b417d716.js");
```

For now we need to keep somewhere a proper name of the file and fetch it before running a worker.

Btw, other javascript files don't have hash in their names. Do we really need to have it only for this specific file?